cli/src/datafeed.ts

```diff
@@ -112,6 +145,73 @@ export function bulkInsertHandler() {
   return wrapper
 }
 
+export function bulkRemoveHandler() {
+  async function wrapper({ data }: { data }) {
+    const bulkData = data as IDatafeedBulk
+
+    if (!checkBulkSource(data?.bulk)) {
+      console.error('invalid json src format')
+      return
+    }
+
+    const adapters = await adapterListHandler()()
+    const aggregators = await aggregatorListHandler()({})
+    const listeners = await listenerListHandler()({})
+    const reporters = await reporterListHandler()({})
+    const delegatorReporters = await delegatorReporterListHandler()()
+    const delegatorContracts = await contractListHandler()()
+    const delegatorFunctions = await functionListHandler()()
+
+    for (const removeElement of bulkData.bulk) {
+      console.log(`removing ${removeElement}`)
+      const adapterData = await loadJsonFromUrl(removeElement.adapterSource)
+      if (!checkAdapterSource(adapterData)) {
+        console.error(`invalid adapterData: ${adapterData}, skipping insert`)
```

Comment:
I think this is coming from the bulk insert.

```suggestion
        console.error(`invalid adapterData: ${adapterData}, skipping removal`)
```